tests/migrations/test_loader.py

```diff
@@ -521,7 +521,12 @@ def test_invalid(self):
         MigrationLoader reraises ImportErrors caused by "bad magic number" pyc
         files with a more helpful message.
         """
-        with self.temporary_migration_module(module='migrations.test_migrations_bad_pyc'):
+        with self.temporary_migration_module(module='migrations.test_migrations_bad_pyc') as migration_dir:
+            # The pyx extension is to avoid the pyc exclusion in MANIFEST.in.
+            os.rename(
+                os.path.join(migration_dir, '0001_initial.pyx'),
```

Comment:
Given that [Pyrex](https://en.wikipedia.org/wiki/Pyrex_(programming_language)) uses `.pyx` this seems an unfortunate name. May I suggest `.pyc-tpl` as we use that elsewhere in the codebase?